### ISSUE TITLE
Iterable fails to build using Carthage

### ISSUE THREAD

**Issue by pepas-everly** [Iterable/swift-sdk#548]
I'm seeing build failures when attempting to use the Iterable SDK via Carthage:

```
/Users/jason.pepas/gitlab/everlywell/ios-app/Everlywell/Carthage/Checkouts/swift-sdk/swift-sdk/swiftui/IterableInboxView.swift:73:27: error: cannot find type 'View' in scope
    public var body: some View {
                          ^~~~
/Users/jason.pepas/gitlab/everlywell/ios-app/Everlywell/Carthage/Checkouts/swift-sdk/swift-sdk/swiftui/IterableInboxView.swift:11:34: error: cannot find type 'View' in scope
public struct IterableInboxView: View {
                                 ^~~~
/Users/jason.pepas/gitlab/everlywell/ios-app/Everlywell/Carthage/Checkouts/swift-sdk/swift-sdk/swiftui/InboxViewRepresentable.swift:22:40: error: cannot find type 'Context' in scope
    func makeUIViewController(context: Context) -> IterableInboxViewController {
                                       ^~~~~~~
/Users/jason.pepas/gitlab/everlywell/ios-app/Everlywell/Carthage/Checkouts/swift-sdk/swift-sdk/swiftui/InboxViewRepresentable.swift:34:91: error: cannot find type 'Context' in scope
    func updateUIViewController(_ uiViewController: IterableInboxViewController, context: Context) {
                                                                                          ^~~~~~~
/Users/jason.pepas/gitlab/everlywell/ios-app/Everlywell/Carthage/Checkouts/swift-sdk/swift-sdk/swiftui/InboxViewRepresentable.swift:11:32: error: cannot find type 'UIViewControllerRepresentable' in scope
struct InboxViewRepresentable: UIViewControllerRepresentable {
                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~

** BUILD FAILED **
```

This is on an M1 mac.

**Comment by pepas-everly** [comment]
Note: as an experiment, I forked this repo and bumped the all of the deployment targets to 13.6, and it then works with Carthage.

(Cartfile entry: `github "pepas-everly/swift-sdk" == 99.2`)

So for some reason, when Carthage builds, it appears the preprocessor directives aren't working correctly.  It attempts to build the SwiftUI code, but fails because the deployment target is below 13.

**Comment by tapashmajumder** [comment]
Should be fixed with version 6.4.2. Please open a ticket with your CSM if you are still seeing issues.
